CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+### New features
+
+### Bug Fixes
+* Fix debug logging deadlock CRT logging from Python logging to avoid deadlock when enabling CRT logs and removed
```

Comment:
Fix deadlock in debug CRT logging when enabling CRT logs. Removed former experimental method _enable_debug_logging()